### DISCUSSION TITLE
Looking for tips to install and configure Frigate on HassOS

### DISCUSSION THREAD

**Discussion by HomeAssistantPim** [blakeblackshear/frigate#14225] [General Support] [answered]
Hi,
I'm interested in using Frigate on my HomeAssistant.
I've got HassOS 12.4 and downloaded the HassOS Addons via HACS (version 2.0.1) as instructed in the frigate documentation and rebooted my ha,
Additionally I have setup Mosquitto MQTT Broker (Addon managed by HA).
Next step seems to add the Frigate Integration and this is were I'm getting stuck with a failed to connect at http://ccab4aaf-frigate:5000
Also I've setup media share named frigate (CIFS 2.0). 
I've added a frigate.yml configuration, not sure if this is needed and what to put in there.
I've added mqtt with a host (ip of my ha) and no user/pwd as this is managed by ha.
Rebooted, still no success.

I've added a database to my config and rebooted, still no luck.
database:
  path: /config/frigate.db

After the download via HACS, in fact the only thing I see is that indeed frigate was downloaded and frigate has become available as an integration in HA.
What part of the installation documentation did I overlook?
What is needed to get the frigate integration installed after downloading the HACS addon and rebooting?
Is there any place were i can see logging from the frigate addon?

**Comment by NickM-27** [comment]
> downloaded the HassOS Addons via HACS (version 2.0.1)

please be mindful of terms. HACS does not install addons, it installs integrations. You need ot install the Frigate addon from the addon store

**Reply by HomeAssistantPim** [reply]
Thanx for the quick reply.
I (think) i did install it from the HACS Addon store?
The normal HA Addons store (where you for instance can also find DuckDNS etc) does not show frigate....

**Reply by NickM-27** [reply]
You don't install addons in HACS. You need to add the frigate repo so it shows up in the addon store like it says in the docs

**Reply by HomeAssistantPim** [reply]
Yes so I downloaded the repo...
Frigate does not show in my HA Addons, it does show up under Integrations which I can't get installed.

**Comment by hawkeye217** [comment]
See the official documentation for installing Frigate (addon and integration) in Home Assistant: https://docs.frigate.video/frigate/installation#home-assistant-addon

**Reply by HomeAssistantPim** [reply]
"Install your desired Frigate NVR Addon and navigate to it's page"

How do I do this on HACS 2.0?

**Reply by hawkeye217** [reply]
You don't do this in HACS. As the docs say:

> 1. Navigate to Supervisor > Add-on Store > Repositories
> 2. Add https://github.com/blakeblackshear/frigate-hass-addons

**Reply by NickM-27** [reply]
You do it in the addon store

**Reply by HomeAssistantPim** [reply]
<img width="897" alt="Schermafbeelding 2024-10-08 221349" src="https://github.com/user-attachments/assets/a6e19e65-1802-4f13-8f0c-a4dfeaae3d30">

**Reply by hawkeye217** [reply, marked as answer]
Click the 3 dots in the top right to find "Repositories"

**Reply by HomeAssistantPim** [reply]
Thnx, I always seem to get lost in the UI when it concerns HACS.
